cloudinary/provisioning/account_config.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+import os
+
+from cloudinary.__init__ import import_django_settings
+from cloudinary.compat import urlparse, parse_qs
+from cloudinary.http_client import HttpClient
+
+
+class AccountConfig(object):
```

Comment:
I don't like this class since it is a 90% duplicate of the `Config` class.. If you can refactor it easily, it would be great, but if it gets complicated, you can keep it like that.